### ISSUE TITLE
Support of ARM

### ISSUE THREAD

**Issue by paulbaudrier** [aws-solutions/instance-scheduler-on-aws#593]
**Describe the feature you'd like**

Currently, the AWS Instance Scheduler Lambda function does not support ARM-based architecture.
We would like the scheduler solution to include support for deploying Lambda functions using the ARM architecture as a parameter. 
In order to enable the leverage of AWS Graviton processors for better cost-effectiveness and performance.

**Comment by CrypticCabub** [comment]
Hi @paulbaudrier,

Could you clarify a little more as to what the ask is here and what the use-case for it would be? are you just wanting to be able to deploy the Instance Scheduler lambdas as ARM functions?

**Comment by paulbaudrier** [comment]
> Hi @paulbaudrier,
> 
> 
> 
> Could you clarify a little more as to what the ask is here and what the use-case for it would be? are you just wanting to be able to deploy the Instance Scheduler lambdas as ARM functions?

Yes we just want to be able to deploy instances scheduler as ARM function.
Enable or disable it through a parameter
@CrypticCabub

**Comment by CrypticCabub** [comment]
For what purpose? Is there a compliance or a cost reason to need to use ARM? Am hoping to understand the use-case a little better before considering yet another configuration parameter on the hub stack

**Comment by paulbaudrier** [comment]
@CrypticCabub For better performance and lower cost of the scheduler

**Comment by whoDoneItAgain** [comment]
any update on this? or has anyone tried switch to ARM? looking at the dependencies very quickly, i don't see anything that would be tied to x86_64
